packages/extension/src/ui/features/accountTokens/AccountTokensButtons.tsx

```diff
@@ -68,6 +70,14 @@ export const AccountTokensButtons: FC<AccountTokensButtonsProps> = ({
       : "wait for this account to deploy"
   } before you can send`
 
+  const showSendButton = useMemo(() => {
+    if (multisig?.needsDeploy) {
+      return false
+    }
+
+    return !!sendToken
```

Comment:
Convention I see in this codebase is to use Boolean() for this
```suggestion
    return Boolean(sendToken)
```